File: rover_motor_controller/rover_motor_controller/vel_parser_node_main.py

import rclpy
from rover_motor_controller.motor_controller import VelParserNode


def main(args=None):
    rclpy.init(args=args)

    node = VelParserNode()

    rclpy.spin(node)

    rclpy.shutdown()


if __name__ == "__main__":
    main()

File: rover_motor_controller_cpp/src/vel_parser_node_main.cpp

#include <memory>

#include "motor_controller/vel_parser_node.hpp"

using namespace motor_controller;

int main(int argc, char *argv[]) {
  rclcpp::init(argc, argv);
  rclcpp::spin(std::make_shared<VelParserNode>());
  rclcpp::shutdown();
  return 0;
}
